pom.xml

```diff
@@ -1399,5 +1407,19 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
```

Comment:
I understand the use case here. But, that is satisfiable without the profiles. I just don't see it feasible to continue adding profiles with new combinations, and I see it confusing to have only a subset of categories defined as profiles, just because a few people wanted to shorten their particular command-line for those particular presets.